packages/react-magma-docs/src/components/docs-heading/index.js

```diff
@@ -43,25 +48,29 @@ const DocsHeading = ({ children, to, type }) => (
       <Heading level={1}>{children}</Heading>
     </div>
     {type === 'api' && to && (
-      <Hyperlink color="secondary" styledAs="Button" to={to}>
-        {linkProps => (
-          <Link {...linkProps}>
-            <PaletteIcon size="16" />
-            <ButtonSpan>View Design Guidelines</ButtonSpan>
-          </Link>
-        )}
-      </Hyperlink>
+      <ButtonContainer>
+        <Hyperlink color="secondary" styledAs="Button" to={to}>
+          {linkProps => (
+            <Link {...linkProps}>
+              <PaletteIcon size="16" />
```

Comment:
Should our icon sizes be coming out of the text scale?

Reply:
Good catch, yes. :)